packages/oraidex-common/src/constant.ts

```diff
@@ -132,16 +132,20 @@ export const WEBSOCKET_RECONNECT_INTERVAL = 20000;
 export const UNISWAP_ROUTER_DEADLINE = 15000; // swap deadline in ms
 export const EVM_BALANCE_RETRY_COUNT = 5;
 
-export const EVM_CHAIN_ID: EvmChainId[] = ["0x38", "0x01", "0x1ae6", "0x2b6653dc"];
-export const COSMOS_CHAIN_ID: CosmosChainId[] = [
-  "Oraichain",
-  "oraibridge-subnet-2",
-  "osmosis-1",
-  "cosmoshub-4",
-  "injective-1",
-  "kawaii_6886-1",
-  "noble-1"
-];
+// evm chainID
+export const ETH_CHAIN_ID = "0x01";
+export const BSC_CHAIN_ID = "0x38";
+export const KAWAII_EVM_CHAIN_ID = "0x1ae6";
+export const TRON_CHAIN_ID = "0x2b6653dc";
+
+// cosmos chainId
+export const ORAICHAIN_CHAIN_ID = "Oraichain";
+export const ORAIBRIDGE_CHAIN_ID = "oraibridge-subnet-2";
+export const OSMOSIS_CHAIN_ID = "osmosis-1";
+export const COSMOSHUB_CHAIN_ID = "cosmoshub-4";
+export const INJECTVE_CHAIN_ID = "injective-1";
+export const KAWAII_COSMOS_CHAIN_ID = "kawaii_6886-1";
+export const NOBLE_CHAIN_ID = "noble-1";
```

Comment:
should using enum instead of const for each chain,
```ts
enum EVM_CHAIN_ID {
...
}
```

Reply:
Yep I agree. Using enum helps with type-safe and groups all chain-ids

Reply:
One problem with changing to enum is that many functions will be affected because I believe we're probably using hard-coded chain id constants everywhere

Reply:
Thanks for this great contribution